config/initializers/assets.rb

```diff
@@ -2,6 +2,13 @@
 #   licensed under the Affero General Public License version 3 or later.  See
 #   the COPYRIGHT file.
 
+# bootstrap-tags plugin relies on rails-assets-bootstrap gem but we use
+# bootstrap-sass this line makes sure we exclude every asset comming
+# from rails-assets-bootstrap to prevent conflicts with bootstrap-sass
+Rails.configuration.assets.paths.reject! do |path|
+  path.include?("rails-assets-bootstrap") && !path.include?("rails-assets-bootstrap-tags")
+end
```

Comment:
And a `require: false` in the `Gemfile` doesn't prevent it from getting loaded? :(